uint8 msgCnt
uint8 id
uint16 secMark
int32 lat
int32 longi
int32 elev
uint8 transmission
uint16 speed
uint16 heading
int16 angle
int16 acc_long
int16 acc_lat
uint8 acc_vert
int16 acc_yaw
uint8 wheelbrakes
uint8 traction
uint8 abs
uint8 scs
uint8 brakeboost
uint8 auxbrakes
st_CAVBSM[] cavbsm
